### ISSUE TITLE
Fails to load configuration during boot

### ISSUE THREAD

**Issue by sanigerson** [vagnum08/cpupower-gui#82]
Hello,
How sad to see that there is no update till 2020. 
I have a problem on latest rasbian os x64.
When i run app and choose any predefined governor, when i restart raspberry, it starts with performance mode.
Is there any solution for this?

**Comment by vagnum08** [comment]
Which version do you have installed?

**Comment by sanigerson** [comment]
Latest official 0.7.2 for raspberry

**Comment by vagnum08** [comment]
I will build a Debian package for raspbian using the latest 1.0.0 for you to install.

Sadly the latest version on the repo is quite old.

**Comment by sanigerson** [comment]
Oh, i'm looking forward to it :)

**Comment by vagnum08** [comment]
I am assuming you are using a RPI 4, hence the x64. Correct?

**Comment by sanigerson** [comment]
Absoultly :)

**Comment by mahmouddalho** [comment]
كيفه الغاء الاشتراك في جفيت

في الخميس، ١٧ مارس ٢٠٢٢ ١١:٠٢ م sanigerson ***@***.***> كتب:

**Comment by vagnum08** [comment]
@sanigerson there is a debian package for raspbian 11 on opensuse build repo

https://download.opensuse.org/repositories/home:/erigas:/cpupower-gui/Raspbian_11/all/cpupower-gui_1.0.0-1_all.deb

**Comment by sanigerson** [comment]
I'm sorry but still same. As you can see, program can't starts also never allows to work cpu over 600mhz

https://youtu.be/KMpEYcW999s

**Comment by vagnum08** [comment]
I see.

Try executing cpupower-gui from a terminal and post the error message here.

I bet it will be a missing dependency, maybe libhandy.

**Comment by sanigerson** [comment]
Here you go:
https://i.hizliresim.com/kvcyhye.png

**Comment by vagnum08** [comment]
That's exactly what I thought.

If you install gir1.2-handy-1 it should work.

**Comment by sanigerson** [comment]
Can you please kindly show me the way how do i should install?

**Comment by vagnum08** [comment]
You could open a terminal and install it by typing or pasting
`sudo apt install gir1.2-handy-1`.

**Comment by sanigerson** [comment]
Waow! You did a magic! :) Will this version be in original repository for raspberry?
Last one thing, any chance that how could i edit profiles on gui?

**Comment by vagnum08** [comment]
The new version would be available when the latest package is accepted on the Debian repos.

To edit the profiles, you edit the setting as you like on the first tab and then you go to the profiles tab, type a name for the profile and then save. The profile will be added on the list.

**Comment by mahmouddalho** [comment]
Yes, this is what happened, but I would like help because I do not know
what service the subscription will be canceled from.

في السبت، ١٩ مارس ٢٠٢٢ ٧:١٢ م vagnum08 ***@***.***> كتب:

**Comment by sanigerson** [comment]
> 

Unfortunatly, it doesn't effect. Still starts up with powersave mode :(

**Comment by vagnum08** [comment]
Have you set the profile on the preferences tab?
![cpg_preferences](https://user-images.githubusercontent.com/16070176/159258151-b12cb180-0610-41a4-97c6-5c79c02d8f69.png)

**Comment by sanigerson** [comment]
Yep, definitly.

Here how raspberry starts:

https://i.hizliresim.com/guhcjek.png

Here's starting profile:
https://i.hizliresim.com/izpplu7.png

And here's loaded profile:
https://i.hizliresim.com/sska61c.png

Also, have you seen my favicons near writing? You can see them on last picture.

**Comment by vagnum08** [comment]
I haven't tested on a raspberrypi yet. I will get a fresh install of the latest Raspbian 64bit OS, install the package and get it working.
I will update accordingly.

**Comment by sanigerson** [comment]
Looking forward to it. Hope it happens very quick :) Thank you.

**Comment by vagnum08** [comment]
![2022-03-29-114049_2560x1080_scrot](https://user-images.githubusercontent.com/16070176/160570978-03c1af37-2399-4a46-8d34-86defa932885.png)


So, in order to get to work you have to change the system-wide configuration file in `/etc/cpupower_gui.conf`.
The settings on the GUI apply to current user session (user's systemd) and the window manager used by the pi is not exacty support by 1.0.0. 

By editing the configuration file in `/etc/` the change is applied at boot by the system's systemd instance, thus it works setting the governor to performance.

Hope it helps!

**Comment by sanigerson** [comment]
Under etc/cpupower-gui, i have no conf. file. If possible, could you please create and share it with me?

**Comment by vagnum08** [comment]
It is `/etc/cpupower_gui.conf`. There is an underscore there instead of a hyphen.

**Comment by vagnum08** [comment]
You have to edit the file as root.
If you are not comfortable at editing from the command line do the following:

First, copy the file to your home folder, e.g.
```
cp /etc/cpupower_gui.conf ~/cpupower_gui.conf
```

Change the file with a text editor. You have to change line 14 (Profile = Balanced).
For the built-in performance profile you change the line to `Profile = Performance`.

Last, you copy the file back as root, i.e. `sudo cp ~/cpupower_gui.conf /etc/cpupower_gui.conf`.

> I believe you won't publish stable version for raspbian os?

I will make the next version to work on raspbian from user settings.
